packages/components/src/components/hive-footer.tsx

```diff
@@ -127,69 +128,73 @@ function List({
   );
 }
 
-const ENTERPRISE: ILink[] = [
-  //   {
-  //     children: 'Consumer Stories',
-  //     href: '#TODO!',
-  //   },
-  //   {
-  //     children: 'Why GraphQL',
-  //     href: '#TODO!',
-  //   },
-  //   {
-  //     children: 'Professional Services',
-  //     href: '#TODO!',
-  //   },
-  //   {
-  //     children: 'Commitment to Security',
-  //     href: '#TODO!',
-  //   },
-];
+export interface HiveFooterItems {
+  developer?: ILink[];
+  company?: ILink[];
+  resources?: ILink[];
+  links?: ILink[];
+}
 
-const DEVELOPER: ILink[] = [
-  {
-    children: 'Documentation',
-    title: 'Read the docs',
-    href: '/docs',
-  },
-  {
-    children: 'Hive Status',
-    title: 'Check Hive status',
-    href: 'https://status.graphql-hive.com/',
-  },
-  {
-    children: 'Hive Updates',
-    title: 'Read most recent developments from Hive',
-    href: 'https://the-guild.dev/graphql/hive/product-updates',
-  },
-  {
-    children: 'Blog',
-    title: 'Read our blog',
-    href: 'https://the-guild.dev/blog',
-  },
-];
+const DEFAULT_ITEMS: HiveFooterItems = {
+  developer: [
+    {
+      children: 'Documentation',
+      title: 'Read the docs',
+      href: '/docs',
+    },
+    {
+      children: 'Hive Status',
+      title: 'Check Hive status',
+      href: 'https://status.graphql-hive.com/',
+    },
+    {
+      children: 'Hive Updates',
+      title: 'Read most recent developments from Hive',
+      href: 'https://the-guild.dev/graphql/hive/product-updates',
+    },
+    {
+      children: 'Blog',
+      title: 'Read our blog',
+      href: 'https://the-guild.dev/blog',
+    },
+  ],
+  company: [
+    {
+      children: 'About',
+      title: 'Learn more about us',
+      href: 'https://the-guild.dev/about-us',
+    },
+    {
+      children: 'Brand Assets',
+      title: 'Brand Assets',
+      href: 'https://the-guild.dev/logos',
+    },
+    {
+      children: 'Newsletter',
+      title: 'Newsletter',
+      href: 'https://the-guild.dev/newsletter',
+    },
+  ],
+  resources: [],
+  links: [
+    {
+      children: 'OSS Friends',
+      href: 'https://the-guild.dev/graphql/hive/oss-friends',
+    },
```

Comment:
@kamilkisiela I made OSS Friends globally shown (because I'm a super friendly guy). The websites can remove it (and possibly Pricing if it doesn't make sense in a context of let's say GraphQL ESLint) by settings `items={{ links: [] }}`